packages/core/src/config/snapwp-config-manager.ts

```diff
@@ -6,13 +6,15 @@ import type { HTMLReactParserOptions } from 'html-react-parser';
 
 export interface SnapWPEnv {
 	/**
-	 * The URL of the Next.js site. Defaults to `process.env.NEXT_PUBLIC_URL`.
+	 * The URL of the Next.js site. Defaults to `process.env.NEXT_PUBLIC_FRONTEND_URL`.
 	 */
-	nextUrl: string;
+	frontendUrl: string;
 	/**
-	 * The home URL of the WordPress site. Defaults to `process.env.NEXT_PUBLIC_WORDPRESS_URL`.
-	 *
-	 * TODO: Update variable name from homeUrl to siteUrl. ref: https://github.com/rtCamp/headless/pull/272#discussion_r1907601796
+	 * The site URL of the WordPress site. Defaults to `process.env.NEXT_PUBLIC_WP_SITE_URL`.
+	 */
+	siteUrl: string;
```

Comment:
As previously flagged, imo `getConfig().siteUrl` and `getConfig().homeUrl` are _not_ self documenting, and should be changed to `wpSiteUrl` and `wpHomeUrl` respectively.

The others are fine, because their names don't conflict with frontend semantics.